apps/vs-code-designer/src/app/commands/workflows/openDesigner/openDesignerForLocalProject.ts

```diff
@@ -166,6 +166,7 @@ export default class OpenDesignerForLocalProject extends OpenDesignerBase {
         this.sendMsgToWebview({
           command: ExtensionCommand.initialize_frame,
           data: {
+            project: 'designer',
```

Comment:
There are const for the project name in here apps/vs-code-react/src/run-service/types.ts

Reply:
I am aware of those project names existing in vs-code-react but was worried for importing that into vs-code-designer. Maybe I am missing something, but I couldn't find const export/imports from vs-code-react folder so it seemed that we cannot import that with @microsoft/vs-code-react. 

project: 'export', 'overview', 'review' also are not using those const for the project name. Would you suggest moving ProjectName const to vscode-extension folder so we can share the const in vs-code-react and vs-code-designer, or would I be missing something to just import the project name from apps/vs-code-react/src/run-service/types.ts into vs-code-designer?

Reply:
yeah if that works we can use the `vscode-extension` lib for that

Reply:
Got it, I'll do that in the next PR!